lib/fastlane/plugin/wpmreleasetoolkit/versioning/formatters/derived_build_code_formatter.rb

```diff
@@ -52,6 +60,10 @@ def build_code(build_code = nil, version:)
         # @raise [StandardError] If the prefix is invalid
         #
         def validate_prefix!(prefix)
+          unless prefix.nil? || prefix.is_a?(String) || prefix.is_a?(Integer)
+            UI.user_error!("Prefix must be a string or integer, got: #{prefix.class}")
+          end
+
           prefix_str = prefix.to_s
```

Comment:
👍 Today I re-learned that `nil.to_s` returns the `''` empty string (as opposed to crashing).

This is why even if now you call `validate_prefix!` from the `def initialize` without doing the `prefix ||= ''` anymore first, this `prefix_str = prefix.to_s` still doesn't crash 👍